email-parser/Cargo.toml

```diff
@@ -35,6 +36,8 @@ compatibility-fixes = []
 content-disposition = ["mime"]
 unrecognized-headers = ["mime"]
 mime = ["textcode"]
+allow-duplicate-headers = []
```

Comment:
I don't think we need a feature for that. I would like to leave the `Email` struct untouched, and rather create a new `PermissiveEmail` struct storing headers as Vec of their values so that it allows duplicate and even missing headers. But `Email` should designate a compliant email.